third-party/cmake/BuildLuarocks.cmake

```diff
@@ -130,6 +130,18 @@ add_custom_target(inspect
 
 list(APPEND THIRD_PARTY_DEPS inspect)
 
+if(USE_BUNDLED_LUA)
+  # DEPENDS on the previous module, because Luarocks breaks if parallel.
+  add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/luabitop
+    COMMAND ${LUAROCKS_BINARY}
+    ARGS build luabitop ${LUAROCKS_BUILDARGS}
+    DEPENDS inspect)
```

Comment:
This is tricky, the `DEPENDS inspect` for penlight (below) could be parallel now.

I guess we need to check `if(USE_BUNDLED_LUA)` below, and set penlight `DEPENDS luabitop` in that case.

Reply:
Or make `luabitop` the last item in the file.  That's probably easier.